fvm/evm/stdlib/contract.cdc

```diff
@@ -457,10 +457,35 @@ contract EVM {
                 )
             }
         }
-        
+
         return ValidationResult(
-        	isValid: true,
-        	problem: nil
+            isValid: true,
+            problem: nil
         )
     }
+
+    /// Block returns information about the latest executed block.
+    access(all)
+    struct EVMBlock {
```

Comment:
Does it need to be `EVMBlock` why not just `Block`, being in EVM contract I think we can conclude it's an EVM block.

Reply:
I first tried that, but `Block` is already taken, because it is defined by Cadence `stdlib`. The same goes for `Address`, that's why the struct here is named `EVMAddress`.

Reply:
Yeah, we intentionally reject type names that potentially shadow built-in types. That way some piece of code can be read and a usage of e.g. `Address` is clear to mean a Flow address, and not e.g. an EVM address.